contracts/script/EnvironmentParser.s.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {Script} from "forge-std/Script.sol";
+import {console} from "forge-std/console.sol";
+import {Vm} from "forge-std/Vm.sol";
+import {StdStyle} from "forge-std/StdStyle.sol";
+
+interface IEigenDADirectory {
+    function getAllNames() external view returns (string[] memory);
+    function getAddress(string memory name) external view returns (address);
+}
+
+interface IOwnable {
+    function owner() external view returns (address);
+}
+
+/// Warning: Assumes the directory correctly contains all contracts for a given network.
+
+contract EnvironmentParser is Script {
+    // ERC1967 storage slots
+    bytes32 constant IMPLEMENTATION_SLOT = 0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc;
+    bytes32 constant ADMIN_SLOT = 0xb53127684a568b3173ae13b9f8a6016e243e63b6e8ee1178d6a717850b5d6103;
```

Comment:
aren't these defined by some OZ library? kinda random that we have to hardcode these like this in here no?